### PR TITLE
fix: svc/job init error out if workload already exists

### LINKED ISSUE #3085: Upgrading from v.1.12 to v1.13 removes any public subnets defined in the env exports
After upgrading from v1.12 to v.1.13 I was left with a broken environment and could not deploy without destroying it and recreating it. 

Breakage was due to the Public Subnets export being removed when being upgraded.

Resolution for me was to tear it all down and recreate the environment with the proper imports. However, this solution isn't sustainable.

### PR DESCRIPTION
**Pull request by iamhopaul123**
fixes #3085

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/job_init.go [hunk 1]
```diff
@@ -254,6 +253,26 @@ func (o *initJobOpts) RecommendActions() error {
 	return nil
 }
 
+func (o *initJobOpts) validateDuplicateJob() error {
+	_, err := o.store.GetJob(o.appName, o.name)
+	if err == nil {
+		log.Errorf(`It seems like you are trying to init a job that already exists.
+To recreate the job, please run:
```
Comment (huanjani) [inline internal/pkg/cli/job_init.go:260]: I wonder if we should mention that the manifest will remain and they can opt to reuse it or not.

Comment (iamhopaul123) [reply internal/pkg/cli/job_init.go:260]: That's a good idea!

File: internal/pkg/cli/svc_init.go [hunk 2]
```diff
@@ -328,6 +332,26 @@ func (o *initSvcOpts) askSvcType() error {
 	return nil
 }
 
+func (o *initSvcOpts) validateDuplicateSvc() error {
+	_, err := o.store.GetService(o.appName, o.name)
+	if err == nil {
+		log.Errorf(`It seems like you are trying to init a service that already exists.
```
Comment (huanjani) [inline internal/pkg/cli/svc_init.go:338]: same as with job, above-- mention manifest file?

**Review by Lou1415926 — APPROVED** [review]

File: internal/pkg/cli/job_init.go [hunk 3]
```diff
@@ -254,6 +253,26 @@ func (o *initJobOpts) RecommendActions() error {
...
+1. %s. Note that the manifest will remain for reusing purpose.
```
Comment (huanjani) [inline internal/pkg/cli/job_init.go]: How about something like, "Note: The manifest file will not be deleted and will be used in Step 2. If you'd prefer a new default manifest, please manually delete the existing one."

**Review by huanjani — APPROVED** [review]
:shipit:

**Pull request merged**